exporter/elasticsearchexporter/model.go

```diff
@@ -294,6 +296,15 @@ func (m *encodeModel) encodeSpanEvent(resource pcommon.Resource, resourceSchemaU
 	otelserializer.SerializeSpanEvent(resource, resourceSchemaURL, scope, scopeSchemaURL, span, spanEvent, idx, buf)
 }
 
+func (m *encodeModel) encodeProfile(resource pcommon.Resource, scope pcommon.InstrumentationScope, record pprofile.Profile, pushData func(*bytes.Buffer, string, string) error) error {
+	switch m.mode {
+	case MappingOTel:
+		return otelserializer.SerializeProfile(resource, scope, record, pushData)
+	default:
+		return errors.New("profiles can only be encoded in OTel mode")
```

Comment:
Perhaps it would help to add a section to README about profiles and mention this and provide an example configuration for profiles?